arbnode/batch_poster.go

```diff
@@ -410,8 +410,10 @@ func NewBatchPoster(ctx context.Context, opts *BatchPosterOpts) (*BatchPoster, e
 	// If the length of the hotshot urls is greater than zero, and it's not length 1 with an empty string, create the espresso multiple nodes client.
 
 	if hotShotUrlsLen != 0 && !(hotShotUrls[0] == "" && hotShotUrlsLen == 1) {
-		//  TODO: tech debt should remove fallback urls in the future
-		hotShotClient := hotshotClient.NewMultipleNodesClient(hotShotUrls)
+		hotShotClient, err := hotshotClient.NewMultipleNodesClient(hotShotUrls)
```

Comment:
This doesn't actually halt the process/return the error when constructing the batch poster

Reply:
log.Crit panics the app if there is an error

Reply:
![image](https://github.com/user-attachments/assets/dec91638-5ea2-4685-a63f-52288d1c1b03)

Reply:
I find that to be very weird behavior from a logging package, but I guess that's fine then.